src/ripple/overlay/impl/PeerImp.cpp

```diff
@@ -1080,8 +1080,7 @@ PeerImp::onMessage (std::shared_ptr <protocol::TMProposeSet> const& m)
 {
     protocol::TMProposeSet& set = *m;
 
-    if (overlay_.setup().expire &&
-            set.has_hops() && ! slot_->cluster())
+    if (set.has_hops() && ! slot_->cluster())
```

Comment:
Thanks @nbougalis and @ximinez.  I guess my real point revolves around a discussion @vinniefalco and I had last weekend -- that we need to improve our documentation.  If this feature is new and experimental, then it's fair for it not to be documented.  But if an auditor were trying to figure out what's going on, they'd be sunk.